README.md

```diff
@@ -1,94 +1,157 @@
-# The ops library
+# The `ops` library
 
-<!-- The text below is also at the top of ops/__init__.py. Keep in sync! -->
 
-The ops library is a Python framework ([`available on PyPI`](https://pypi.org/project/ops/)) for developing
-and testing [Juju](https://juju.is/) charms in a consistent way, using standard Python constructs
-to allow for clean, maintainable, and reusable code.
+The `ops` library is a Python framework for developing and testing Kubernetes and machine charms. While charms can be written in any language, `ops` defines the latest standard, and all new charms are encouraged to use Python with `ops`. The library is an official component of the Charm SDK, itself a part of [the Juju universe](https://juju.is/).
 
-A charm is an operator -- business logic encapsulated in a reusable software
-package that automates every aspect of an application's life.
+> - `ops` is  [available on PyPI](https://pypi.org/project/ops/).
+> - The latest version of `ops` requires Python 3.8 or above.
 
-Charms written with ops support Kubernetes using Juju's "sidecar charm"
-pattern, as well as charms that deploy to Linux-based machines and containers.
+||||
+|-|-|- |
+|| [Juju](https://juju.is/docs/juju) | Learn how to quickly deploy, integrate, and manage charms on any cloud with Juju. <br>  _It's as simple as `juju deploy foo`, `juju integrate foo bar`, ..., on any cloud._ |
+||||
+|| [Charmhub](https://charmhub.io/) | Sample our existing charms on Charmhub. <br> _A charm can be a cluster ([OpenStack](https://charmhub.io/openstack-base), [Kubernetes](https://charmhub.io/charmed-kubernetes)), a data platform ([PostgreSQL](https://charmhub.io/postgresql-k8s), [MongoDB](https://charmhub.io/mongodb), etc.), an observability stack ([Canonical Observability Stack](https://charmhub.io/cos-lite)), an MLOps solution ([Kubeflow](https://charmhub.io/kubeflow)), and so much more._ |
+||||
+|:point_right:| [Charm SDK](https://juju.is/docs/sdk) | Write your own charm! <br> _Juju is written in Go, but our SDK supports easy charm development in Python._  |
```

Comment:
Non-breaking space in "Charm SDK" so it doesn't wrap onto two lines in the output:

```suggestion
|:point_right:| [Charm SDK](https://juju.is/docs/sdk) | Write your own charm! <br> _Juju is written in Go, but our SDK supports easy charm development in Python._  |
```

Reply:
OK